crates/tuono/src/scaffold_project.rs

```diff
@@ -66,9 +67,18 @@ pub fn create_new_project(
     folder_name: Option<String>,
     template: Option<String>,
     select_head: Option<bool>,
+    git: Option<bool>,
 ) {
     let folder = folder_name.unwrap_or(".".to_string());
 
+    // Check if Git is installed when the user requests to use it; otherwise, continue
+    if git.unwrap_or(false) && !is_git_installed() {
+        exit_with_error("You requested to use Git, but it is not installed.")
+    }
+
+    // Use git by default
+    let git = git.unwrap_or(true) && is_git_installed();
```

Comment:
I'd avoid running the command twice

```suggestion
    let is_git_installed = is_git_installed();

    // Check if Git is installed when the user requests to use it; otherwise, continue
    if git.unwrap_or(false) && !is_git_installed {
        exit_with_error("You requested to use Git, but it is not installed.")
    }

    // Use git by default
    let git = git.unwrap_or(true) && is_git_installed;
```